hoomd/mpcd/ConcentricCylindersGeometry.cc

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2009-2024 The Regents of the University of Michigan.
+// Part of HOOMD-blue, released under the BSD 3-Clause License.
+
+/*!
+ * \file mpcd/ConcentricCylindersGeometry.cc
+ * \brief Export function MPCD concentric cylinders geometry.
+ */
+
+#include "ConcentricCylindersGeometry.h"
+
+namespace hoomd
+    {
+namespace mpcd
+    {
+namespace detail
+    {
+void export_ConcentricCylindersGeometry(pybind11::module& m)
+    {
+    pybind11::class_<ConcentricCylindersGeometry, std::shared_ptr<ConcentricCylindersGeometry>>(
+        m,
+        ConcentricCylindersGeometry::getName().c_str())
+        .def(pybind11::init<Scalar, Scalar, Scalar, bool>())
+        .def_property_readonly("R0", &ConcentricCylindersGeometry::getR0)
+        .def_property_readonly("R1", &ConcentricCylindersGeometry::getR1)
+        .def_property_readonly("speed", &ConcentricCylindersGeometry::getSpeed)
```

Comment:
We prefer to use descriptive variable names in Python, so I would propose the following:
```suggestion
        .def_property_readonly("inner_radius", &ConcentricCylindersGeometry::getR0)
        .def_property_readonly("outer_radius", &ConcentricCylindersGeometry::getR1)
        .def_property_readonly("angular_speed", &ConcentricCylindersGeometry::getSpeed)
```
You might want to rename the getters in C++ accordingly. It is fine to use `R0`, etc. internally in the class if you prefer the shorter variable names there.